content/docs/capabilities/routing.mdx

```diff
@@ -407,6 +407,23 @@ routes:
       - http://myapp-srv-5:8080,10
 ```
 
+## Route matching order
+
+If your configuration includes multiple routes with identical definitions, Pomerium will process these routes in the order they are defined in your configuration file.
```

Comment:
pomerium configuration may be sourced from config file and databroker configuration. 

anything defined in the config file is processed ahead of whatever comes from the databroker, and yes it is processed in the order as it's listed in the config file. 

routes that are defined in the databroker, which is the case for the ingress controller, enterprise console and zero, are sorted in the deterministic manner: 

1. Ascending by `host`.
2. Descending by `path`.
3. Descending by `regex`.
4. Descending by `prefix`.